geps/gep-1494/index.md

```diff
@@ -0,0 +1,151 @@
+
+Gateway API already has some work in progress to handle this for straightforward use cases, in [GEP-91](https://gateway-api.sigs.k8s.io/geps/gep-91/).
+
+#### JWT
+
+JWT is a format for representing _claims_, which are stored as fields in a JSON object. In common use, these claims represent various parameters that describe the properties of an authentication, with sample claims being `iss`, Issuer, `sub`, Subject, and `aud`, Audience.
+
+JWT also specified ways in which JWTs can be _nested_, where a JWT is encoded inside another JWT, which allows the use of encryption and signing algorithms. In the Gateway API Auth* case, this is expected to be common, in a pattern looking something like this:
+
+- a common authority issues keypairs to clients and servers
+- clients include JWTs with requests that identify themselves and their intended uses
+- servers unwrap encryption and validate signatures of these JWTs to validate the chain of trust
+- servers use the authentication details to make authorization decisions.
+
+JWT is defined in [RFC-75199](https://datatracker.ietf.org/doc/html/rfc7519).
+
+#### Oauth2 and OIDC
+
+Oauth2 is an _authorization framework_, which allows clients and servers to define ways to perform authentication and authorization in as secure a way as possible. It extensively uses TLS for encryption, and involves a third-party handling the authorization handshake with a client, which the third-party then provides to the server.
+
+Open ID Conect (OIDC) is a protocol based on the OAuth 2 framework, that allows Users to talk to Identity Providers (IDPs), on behalf of a Relying Party (RP), and have the IDP give the user an Identity Token (which the User's browser can then provide as Authentication to the Relying Party), and also allows the RP to request Claims about the User, which can be used for Authorization.
+
+
+## Auth* User Stories
+
+
+* As Ana the Application Developer, I wish to be able to configure that some or all of my service exposed via Gateway API requires Authentication, and ideally to be able to make Authorization decisions about _which_ authenticated clients are allowed to access.
+* As Ana the Application Developer, I wish to be able to redirect users to a login page when they lack authentication, while unauthenticated API access gets the proper 40x response.
+* As Chihiro the Cluster Admin, I wish to be able to configure default Authentication settings (at least), with an option to enforce Authentication settings (preferable but not required) for some set of services exposed via Gateway API inside my cluster.
+* More User Stories welcomed here!
+
+## Currently implemented Auth mechanisms in implementations
+
+Many Gateway API implementations have implemented some method of configuring Auth* that are currently Implementation Specific.
```

Comment:
At least for the Istio examples, these aren't actually using Gateway API implementation-specific extensibility mechanisms (HTTPRoute filters, etc), but rather custom annotations and CRDs. It may be worth clarifying this statement a bit to better reflect if these are intended to be Gateway API well-known or ad-hoc extension examples, or just capabilities of implementations which also support Gateway API configuration.